extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -509,7 +509,8 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
    * @private
    */
   getPublisherOrigin_() {
-    const publisherOrigin = getSourceOrigin(this.getAmpDoc().getUrl());
-    return publisherOrigin.replace(/https?:\/\//, '');
+    let publisherOrigin = getSourceOrigin(this.getAmpDoc().getUrl());
+    publisherOrigin = publisherOrigin.replace(/https?:\/\//, '');
+    return publisherOrigin.replace(/^www./, '');
```

Comment:
Can we do it in one replace? I'm really bad at this but `https?:\/\/(www.)?`?

Reply:
We probably can, but I'm also less confident about writing a single regex that works as surely as two `replace()` calls without accidentally introducing a bug

Reply:
It's okay to have bugs sometimes, I think we can test this fairly well with online regex tools